e2e/cypress/integration/onboarding/enter_support_email_spec.js

```diff
@@ -7,15 +7,13 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Group: @enterprise @onboarding @cloud_only
+// Group: @onboarding
```

Comment:
All tests failed on me here. What's the requirement/flow to show up "Enter support email" step?
<img width="736" alt="Screen Shot 2021-08-16 at 12 53 57 PM" src="https://user-images.githubusercontent.com/5334504/129520975-da15b822-0c0e-402d-a94c-816afc361764.png">